web-marketplace/src/components/organisms/AgreePurchaseForm/AgreePurchaseForm.tsx

```diff
@@ -150,7 +150,7 @@ export const AgreePurchaseForm = ({
           {...form.register('agreeErpa')}
         />
       </div>
-      <div className="float-right pt-40">
+      <div className="float-right pt-40 sm:pt-60">
```

Comment:
I know it is not a new introduction, but we should probably not use `float-right` as it's better to use `flex justify-end`, for example.

```suggestion
      <div className="flex justify-end pt-40 sm:pt-60">
```

Same would apply to the `<PrevNextButtons>` wrapper `div` in `ChooseCreditsForm.tsx` and `PaymentInfoForm.tsx`.

Reply:
updated